build.gradle

```diff
@@ -91,22 +91,25 @@ ext {
   androidXLifecycleViewmodel = "androidx.lifecycle:lifecycle-viewmodel:$lifecycleVersion"
   androidXLifecycleViewmodelKtx = "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycleVersion"
 
+  threeTenAbpVersion = "1.4.3"
```

Comment:
i noticed that we had two separate junit versions due to versions being updated in separate PR and thought this might be the issue. i created a property for libraries that should maintain the same version to avoid this in the future